src/source/canvas_source.js

```diff
@@ -135,6 +135,8 @@ class CanvasSource extends ImageSource {
 
         if (Object.keys(this.tiles).length === 0) return; // not enough data for current position
 
+        if (!this._playing && this.texture) return; // Already prepared the static image
```

Comment:
Hmm, actually, this is wrong -- `_prepareImage` needs to be called if the coordinates have changed, to re-initialize `boundsBuffer`.